src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -194,11 +193,11 @@ public void postUsers(String lang, User entity,
           Criteria idCrit = new Criteria();
           idCrit.addField(USER_ID_FIELD);
           idCrit.setOperation("=");
-          idCrit.setValue(entity.getId());
+          idCrit.setVal(entity.getId());
           Criteria nameCrit = new Criteria();
           nameCrit.addField(USER_NAME_FIELD);
           nameCrit.setOperation("=");
-          nameCrit.setValue(entity.getUsername());
+          nameCrit.setVal(entity.getUsername());
           Criterion crit = new Criterion();
           crit.addCriterion(idCrit, "OR", nameCrit);
```

Comment:
These checks are not needed: Both id and username have a unique index and the insert will automatically fail on insert. Can you try to remove this duplicated code?